backend/notifications/registry.py

```
@@ -0,0 +1,63 @@
+
+EVENT_TYPES = {event.slug: event for event in _EVENT_TYPES}
+
+
+def get_event_type(slug):
+    try:
+        return EVENT_TYPES[slug]
+    except KeyError:
+        raise KeyError(
+            f"Unknown notification event type '{slug}'. "
+            "Register it in notifications/registry.py."
+        )
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)